### ISSUE TITLE
[URGENT] POC NEEDED

### ISSUE THREAD

**Issue by MohabEG** [123FLO321/POGOProtos-Swift#48]
Can you add how it works section?

**Comment by Furtif** [comment]
> Can you add how it works section?

??????

**Comment by Fabio1988** [comment]
> Can you add how it works section?

What about "pls"? What do you want to achieve? You want to use this module? You want to update, or what? :)

**Comment by MohabEG** [comment]
@Furtif @Fabio1988, I am sorry, I didnt mean to be mean

I need to know how this Proto works for pokemon go and can it show nearby pokemons?
and why is the file so big 12MB my xcode, I am willing to help by breaking down this big file to smaller files and make your repo better but how does it work firstly?
Can someone explain please?

**Comment by Fabio1988** [comment]
This is just a communication protocol dump, this can't show nearby pokemon. How did you find this repository? What are you trying to achieve?

**Comment by MohabEG** [comment]
@Fabio1988 if I wanna see nearby pokemons, can it be achieved with this protocol?

**Comment by Fabio1988** [comment]
> @Fabio1988 if I wanna see nearby pokemons, can it be achieved with this protocol?

No, you need the game, so download it from PlayStore/AppStore ;) 
If you are asking these questions this repo is the wrong one for you.

So again, answer my questions from above ;)

**Comment by MohabEG** [comment]
@Fabio1988 Does this repo https://github.com/RealDeviceMap/RealDeviceMap allow me to see nearby pokemons?

**Comment by Fabio1988** [comment]
> @Fabio1988 Does this repo https://github.com/RealDeviceMap/RealDeviceMap allow me to see nearby pokemons?

Your answers are too vague to me, find it out ;)
